ipatests/util.py

```diff
@@ -25,15 +25,15 @@
 
 import inspect
 import os
-from os import path
-import tempfile
-import shutil
 import re
+import shutil
+import tempfile
 import uuid
-import pytest
 from contextlib import contextmanager
+from os import path
 from pprint import pformat
 
+import pytest
```

Comment:
I assume these import changes aren't needed anymore?

Reply:
It is unused in the code, but I am not sure any more. Nothing is certain. Entropy is over 900.